executor/executor.go

```diff
@@ -63,6 +66,16 @@ func New(sdAPI api.API, args []string) (Executor, error) {
 
 func execCommand(path string, args []string) (err error) {
 	cmd := command(path, args...)
+	if !terminal.IsTerminal(0) {
+		stdin, err := cmd.StdinPipe()
+		if nil != err {
+			lgr.Debug.Printf("failed to open StdinPipe: %v", err)
+		}
+		b, _ := ioutil.ReadAll(os.Stdin)
```

Comment:
I think it is better not to ignore error.

Reply:
Added error handle.